.github/matrix-full.yml

```diff
@@ -78,11 +87,12 @@ build-images:
 
 # Ubuntu
 - label: ubuntu
-  base-image: ubuntu:22.04
+  base-image: ubuntu:24.04
   package: deb
-  artifact-from: ubuntu-22.04
-  artifact-from-alt: ubuntu-22.04-arm64
+  artifact-from: ubuntu-24.04
+  artifact-from-alt: ubuntu-24.04-arm64
   docker-platforms: linux/amd64, linux/arm64
+  check-manifest-suite: docker-image-ubuntu-24.04
```

Comment:
this might bring more deviatiom in ubuntu26.04 or debian if they choose to do so. we can let explain_manifest to optionally to read in the `/etc/passwd` file so we can test on username directly instead of UID/GID.
not a blocker for now, feel free to ignore me. i can look at this later after this is merged

Reply:
Yeah.. matching on username alone would be a good way to tackle this.

The better way to solve this type of problem would be to pick a single UID (and GID) that fits nicely into the expected ranges for regular, service, or applications users on all the platforms that we support.